README.md

```diff
@@ -1 +1,194 @@
-# book-an-appointment
\ No newline at end of file
+<a name="readme-top"></a>
```

Comment:
- [ ] There are some tests that are not passing on my end:

![image](https://github.com/MenendezON/book-an-appointment/assets/57421823/3289cb19-7587-4da6-9354-ba96b3accea2)

Kindly, make sure all your tests pass in order to preserve your app's integrity.; 👍